tests/link/pytorch/test_slinalg.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import pytest
+
+import pytensor
+from pytensor.tensor import tensor
+from pytensor.tensor.slinalg import cholesky
+
+
+torch = pytest.importorskip("torch")
+
+
+# @todo: We don't have blockwise yet for torch
```

Comment:
This is a specific test, just test cholesky and solve directly with the `compare_py_and_torch` as usual